python/main.py

```diff
@@ -1,14 +1,23 @@
+import sys
+
 from sidecar import chat, cli, ingest, rewrite
 
 if __name__ == '__main__':
     parser = cli.get_arg_parser()
     args = parser.parse_args()
+
     if args.debug:
         print(args)
+    
     if args.command == "ingest":
+        if args.status:
+            ingest.get_statuses()
+            sys.exit()
```

Comment:
I don't think that implementing this a an option of ingest would work with the CliCommands typings we are using. We can only vary the output type by command and not by parameters.

Also, you need to update the README.md to add this new command.